api/src/gcnotfiy-services/gcnotify-service.ts

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios';
+import { ApiError, ApiErrorType } from '../errors/custom-error';
+import { IgcNotfiyPostReturn, IgcNotfiyGenericMessage } from '../models/gcnotify';
+
+export class GCNotifyService {
+  /**
+   * Sends api call to gcnotify to send notification
+   *
+   *
+   * @param {string} url
+   * @param {object} config
+   * @param {object} data
+   */
+  sendGCNotification = async (url: string, config: object, data: object): Promise<IgcNotfiyPostReturn> => {
```

Comment:
Looks like this function is never used currently.  Probably best to remove it for now in that case.